.pre-commit-config.yaml

```diff
@@ -99,6 +100,7 @@ repos:
         pass_filenames: false
 
   - repo: https://github.com/ansible-network/collection_prep
```

Comment:
This should get moved to the top so that generated files get linted by later hooks

Reply:
done, thx